src/main/resources/com/google/api/codegen/go/main.snip

```diff
@@ -116,7 +116,7 @@
     // use by Google-written clients.
     func (c *{@view.clientTypeName}) SetGoogleClientInfo(name, version string) {
         goVersion := strings.Replace(runtime.Version(), " ", "_", -1)
-        v := fmt.Sprintf("%s/%s %s gax/%s go/%s", name, version, gapicNameVersion, gax.Version, goVersion)
+        v := fmt.Sprintf("%s/%s %s gax/%s gl-go/%s", name, version, gapicNameVersion, gax.Version, goVersion)
```

Comment:
This is the heart of the change, so I have several comments here:

- The first tuple contains `gax/{version}`, which is duplicative.
- How can a Veneer override `gapic/{version}` with `gccl/{version}`, so that we can tell when a Veneer is used vs. the direct GAPIC surface?
- The GAPIC version is not required when the `gccl` version is defined, since their relationship is deterministic (assuming we keep track of the version of GAPIC shipped with each Veneer version).
- Do you have access to the gRPC version? Even though it's reported in `user-agent`, the Chemist team does not want to look for it there. It should be reported in this custom header as `grpc/{version}`.
- Please see Luke's email today about possibly changing the ordering, and weigh in if you have opinions. Since you hard-code the string, I imagine it's all the same to you.